SQLite.CodeFirst/Internal/Builder/CreateTableStatementBuilder.cs

```diff
@@ -19,13 +21,19 @@ public CreateTableStatementBuilder(EntitySet entitySet, AssociationTypeContainer
 
         public CreateTableStatement BuildStatement()
         {
+            var nonAutoincrementKeys = entitySet.ElementType.KeyMembers
+                .Where(km => (km as EdmProperty)?.GetCustomAnnotation<AutoincrementAttribute>() == null);
```

Comment:
Actually I learned that AUTOINCREMENT does also work without a PK:
https://www.tutorialspoint.com/sqlite/sqlite_using_autoincrement.htm

So I guess the constraint is the right approach but not in combination with the primary key.
I'll investigate it further and come up with a solution.